templates/kselftest/kselftest.jinja2

```diff
@@ -0,0 +1,25 @@
+- test:
+    timeout:
+      minutes: 55
+
+    definitions:
+    - from: inline
+      repository:
+        metadata:
+          format: Lava-Test Test Definition 1.0
+          name: timesync-off
+          description: Disable systemd time sync services
+        run:
+          steps:
+          - systemctl stop systemd-timesyncd || true
+      name: timesync-off
+      path: inline/timesync-off.yaml
+
+    - repository: https://github.com/Linaro/test-definitions.git
+      from: git
+      revision: master
```

Comment:
One issue with using the head of the master branch is that it makes it harder to reproduce results.  There doesn't seem to be any regular release tags in the `test-definitions` repository so we should either ask for tags to be created or keep a mirror and add KernelCI tags for each production update.

Reply:
As discussed, we will need a fork of `test-definitions` for KernelCI in any case to have a staging branch.  We could use that fork with a `kernelci.org` branch used in production too.